tests/parser/test_recipe_parser.py

```diff
@@ -1,22 +1,18 @@
 """
 File:           test_recipe_parser.py
-Description:    Unit tests for the recipe parser class and tools
+Description:    Unit tests for the RecipeParser class
 """
 from __future__ import annotations
 
-from pathlib import Path
 from typing import Final
 
 import pytest
 
 from conda_recipe_manager.parser.enums import SelectorConflictMode
 from conda_recipe_manager.parser.exceptions import JsonPatchValidationException
 from conda_recipe_manager.parser.recipe_parser import RecipeParser
-from conda_recipe_manager.parser.types import MessageCategory
 from conda_recipe_manager.types import JsonType
-
-# Path to supplementary files used in test cases
-TEST_FILES_PATH: Final[str] = "tests/test_aux_files"
+from tests.conftest import TEST_FILES_PATH, load_file, load_recipe
```

Comment:
Pytest conftest modules shouldn't be imported directly, it's better to move that into a tests/file_loading.py or something that you can import, or define it as a fixture that takes request parameters via the ["Indirect parametrization" feature](https://docs.pytest.org/en/7.1.x/example/parametrize.html#indirect-parametrization).

Reply:
I'll just rename the file for now, to keep it simple. I think we have other projects in Distro that mis-use `conftest.py` in that case.